nilearn/datasets/utils.py

```diff
@@ -62,6 +62,17 @@ def readlinkabs(link):
         return path
     return os.path.join(os.path.dirname(link), path)
 
+def pheno_decode(a):
```

Comment:
I think that I would call this "char_array_decode", or something like this, rather than a reference to phenotypes, which is not clear out of context.